json_merger/comparator.py

```diff
@@ -29,8 +29,17 @@
 
 
 class BaseComparator(object):
+    """Abstract base class for Entity Comparison."""
 
     def __init__(self, l1, l2):
+        """
+        Do not override the constructor since it will be called by the
+        json_merger logic.
```

Comment:
I mean... docstrings at this point are the only thing I can review. For the rest I assumes tests and APIs are the final source of truth.

Gaaah :boom: